Sources/FINNSetup/FilterMarkets/FilterMarketRealestate.swift

```diff
@@ -270,7 +270,7 @@ extension FilterMarketRealestate: FINNFilterConfiguration {
 
         switch filterKey {
         case .noOfBedrooms:
-            return StepperFilterConfiguration(minimumValue: 0, maximumValue: 6, unit: "stk")
+            return StepperFilterConfiguration(minimumValue: 0, maximumValue: 6, unit: "stk", alternativeUnit: "soverom")
```

Comment:
Why don't we use "stk" for bedrooms?

Reply:
I asked on Trello, I hope they say we can use "stk".

Reply:
Now it only has the `+` at the end.